.github/workflows/test_exporters_common.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          pip install --no-cache-dir torch==2.1.2 torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Comment:
removing this pin result in longer installation time, because we install latest cpu wheels and then in the next line the resolver uninstalls it to install torch 2.1.2 from pypi (with cuda wheels)